docs/source/reservations/reservations.rst

```diff
@@ -0,0 +1,182 @@
+
+.. _reservation:
+
+Reservations & DWS
+==================
+
+
+Clouds are designed to be elastic, but due to the recent GPU shortages, many organizations have started to get reservations
+from cloud providers to lock in GPU capacity.
+
+SkyPilot can be used to request resources from reservations and even combine them with on-demand/spot resources to fully
+utilize the capacity in your cloud accounts.
+
+AWS capacity reservations & capacity block
+------------------------------------------
+
+AWS capacity reservations and capacity block offers are a way to lock in a certain amount of compute capacity for a period of time. The latter is for high-end GPUs, such as A100s(P4d instances) and H100s(P5d instances), while the former is for all other instance types.
+Instead of committing to a 1-3 year long contract, you can get a capacity reservation and capacity block for as short as 1 second and 1 day, respectively.
+
+
+To request capacity reservations/blocks, see the official docs:
+
+* `AWS Capacity Reservations <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-capacity-reservations.html>`_
+* `AWS Capacity Block Offers <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-capacity-blocks.html>`_
+
+Once you have got a reservation/block, you will get an id of your reservation, like ``cr-0123456789abcdefg``.
+
+You can specify two fields in ``~/.sky/config.yaml``:
+
+* ``aws.prioritize_reservations``: whether to prioritize launching clusters from capacity reservations in any region/zone over on-demand/spot clusters. This is useful to fully utilize your reserved capacity created with *Instance eligibility: open*.
+* ``aws.specific_reservations``: a list of reservation ids that can be used by SkyPilot. This is useful if you have multiple capacity blocks or capacity reservations with *Instance eligibility: targeted* for different instance types in multiple regions/zones.
+
+.. note::
+
+    If any of the fields are specified, please allow about 30 seconds for SkyPilot optimizer to retrieve the latest reservation/block status on all regions and zones from your AWS account.
+
+.. code-block:: yaml
+
+    aws:
+      prioritize_reservations: true
+      specific_reservations:
+        # 1x H100 capacity block in us-east-1
+        - "cr-0123456789"
+        # 2x A100 reservation in us-west-2
+        - "cr-123456789a"
+        # 2x M5a.16xlarge reservation in us-east-1
+        - "cr-23456789ab"
+
+For more details of the configuration, see reference for :ref:`config-yaml`.
+
+
+
+.. _utilizing-reservations:
+
+Utilizing Reservations
+~~~~~~~~~~~~~~~~~~~~~~
+
+By specifying configurations above, SkyPilot will automatically figure out the best place to launch your clusters with available capacity in your reservations/blocks considered as zero-cost.
```

Comment:
```suggestion
By specifying the two fields above, SkyPilot will consider available capacity in your reservations/blocks as zero cost and automatically figure out the best place to launch your clusters.
```
Can we replace this to an even more intuitive sentence? Like "...will prioritize using any available reservation/block (i.e., consider them as zero cost) whenever you launch a cluster/job."

Reply:
Good point! Updated.